### DISCUSSION TITLE
How to close some rules? How to ignore for next line or whole file?

### DISCUSSION THREAD

**Discussion by adoin** [oxc-project/oxc#1669] [Q&A] [answered]
Any document about this?

**Comment by Boshen** [comment]
Thank you for trying out oxlint.

> How to close some rules

please let me know if `npx oxlint@latest --help` is unclear.

`npx oxlint@latest -D correctness -A rule-name-1 -A rule-name-2` should do the trick.

Which rules are you trying to disable> Perhaps they need to be modified.

**Reply by adoin** [reply]
I want disable "no-useless-fallback-in-spread"
I have a lot of code like this`defHttp.post({
    url,
    params: {
      pageSize: page.pageSize,
      pageNum: page.currentPage,
      ...(form ?? {}),
    },
  })`

**Reply by adoin** [reply]
key point is I hope oxc can read config file on project root which defined ignore rule or else, is that possible?

**Comment by Boshen** [comment]
> How to ignore for next line or whole file

Please let me know if eslint comment disabling https://eslint.org/docs/latest/use/configure/rules#disabling-rules doesn't work.

**Reply by adoin** [reply]
Clear now.

**Comment by Boshen** [comment, marked as answer]
> Any document about this?

Please let me know if https://oxc-project.github.io/docs/guide/usage/linter.html is inaccessible or unclear to you.
